package.json

```diff
@@ -52,14 +52,6 @@
     "looks-same": "^9.0.1",
     "sharp": "^0.34.3"
   },
-  "peerDependencies": {
-    "tscircuit": "*"
-  },
-  "peerDependenciesMeta": {
-    "tscircuit": {
-      "optional": false
```

Comment:
can you set `optional: true` instead? I'm pretty sure this change will cause tscircuit to be bundled